docs/getting-started.md

```diff
@@ -67,20 +67,93 @@ KONNECTD_DEBUG_TOKEN
 : Token to grant metrics access, empty default value
 
 KONNECTD_DEBUG_PPROF
-: Enable pprof debugging, defaults to `false`
+: Enable pprof debugging
 
 KONNECTD_DEBUG_ZPAGES
-: Enable zpages debugging, defaults to `false`
+: Enable zpages debugging
 
 KONNECTD_HTTP_ADDR
 : Address to bind http server, defaults to `0.0.0.0:9130`
 
-KONNECTD_HTTP_NAMESPACE
-: The http namespace
-
 KONNECTD_HTTP_ROOT
 : Root path of http server, defaults to `/`
 
+KONNECTD_HTTP_NAMESPACE
+: Set the base namespace for service discovery, defaults to `com.owncloud.web`
+
+KONNECTD_IDENTITY_MANAGER
+: Identity manager (one of ldap,kc,cookie,dummy), defaults to `ldap`
+
+KONNECTD_TRANSPORT_TLS_CERT
+: Certificate file for transport encryption, empty default value
+
+KONNECTD_TRANSPORT_TLS_KEY
+: Secret file for transport encryption, empty default value
+
+KONNECTD_ISS
+: OIDC issuer URL, defaults to `https://localhost:9130`
+
+KONNECTD_SIGNING_PRIVATE_KEY
+: Full path to PEM encoded private key file (must match the --signing-method algorithm)
+
+KONNECTD_SIGNING_KID
+: Value of kid field to use in created tokens (uniquely identifying the signing-private-key), empty default value
+
+KONNECTD_VALIDATION_KEYS_PATH
+: Full path to a folder containg PEM encoded private or public key files used for token validaton (file name without extension is used as kid), empty default value
+
+KONNECTD_ENCRYPTION_SECRET
+: Full path to a file containing a %d bytes secret key, empty default value
+
+KONNECTD_SIGNING_METHOD
+: JWT default signing method, defaults to `PS256`
+
+KONNECTD_URI_BASE_PATH
+: Custom base path for URI endpoints, empty default value
+
+KONNECTD_SIGN_IN_URI
+: Custom redirection URI to sign-in form, empty default value
+
+KONNECTD_SIGN_OUT_URI
+: Custom redirection URI to signed-out goodbye page, empty default value
+
+KONNECTD_ENDPOINT_URI
+: Custom authorization endpoint URI, empty default value
+
+KONNECTD_ENDSESSION_ENDPOINT_URI
+: Custom endsession endpoint URI, empty default value
+
+KONNECTD_ASSET_PATH
+: Path to custom assets, empty default value
+
+KONNECTD_IDENTIFIER_CLIENT_PATH
+: Path to the identifier web client base folder, defaults to `/var/tmp/konnectd`
+
+KONNECTD_IDENTIFIER_REGISTRATION_CONF
+: Path to a identifier-registration.yaml configuration file, defaults to `./config/identifier-registration.yaml`
+
+KONNECTD_IDENTIFIER_SCOPES_CONF
+: Path to a scopes.yaml configuration file, empty default value
+
+KONNECTD_INSECURE
+: Disable TLS certificate and hostname validation
+
+KONNECTD_TLS
+: Use TLS (disable only if konnectd is behind a TLS-terminating reverse-proxy), defaults to `true`
+
+KONNECTD_TRUSTED_PROXY
+: Trusted proxy IP or IP network (can be used multiple times)
```

Comment:
I would reword:

> can be used multiple times

to a usage example, as it's not really POSIX. Something like:

> List of trusted proxy IP or IP network (usage: --trusted-proxy x.x.x.x --trusted-proxy y.y.y.y)

This way the consumer knows it's a list, and not overwriting values.

Reply:
same for the other occurrences of this wording.